core/modules/views/templates/views.theme.inc

```diff
@@ -930,10 +930,14 @@ function template_preprocess_views_view_list(&$variables) {
   $handler  = $variables['view']->style_plugin;
 
   $class = explode(' ', $handler->options['class']);
-  $class = array_map('backdrop_clean_css_identifier', $class, array(array()));
+  foreach ($class as $key => $individual) {
```

Comment:
Wondering if we can rename the `$class` variable above this line to `$classes` instead? That way, we could do this, which makes more sense to me:
```suggestion
  foreach ($classes as $key => $class) {
```